### PR TITLE
Add possibility for metrics from the zap logger

### PR DESCRIPTION
**Pull request by drPytho**

### CODE REVIEW

File: logger.go [hunk 1]
```diff
@@ -4,22 +4,130 @@ import (
...
 	), time.Second, 100, 10),
-		zap.ErrorOutput(zapcore.Lock(os.Stderr)),
-		zap.AddCaller(),
+		zapOpts...,
 	)
 
 	return logger, atom
 }
+
+func WithZapMetrics() Option {
```
Comment (djui) [inline logger.go:30]: Add a func comment. #love-oss

File: logger.go [hunk 2]
```diff
@@ -4,22 +4,130 @@ import (
...
+	return func(s *SVC) error {
+		requestCount := prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "logger_emitted_entries",
+				Help: "Number of log messaged emitted.",
```
Comment (djui) [inline logger.go:35]: So close. Typo: `messages`

Comment (drPytho) [reply logger.go]: And we even have the misspell linter enabled. :thinking:

File: logger.go (hunk 1 above)
Comment (drPytho) [reply logger.go:32]: Hope this is a nice comment :haircut_man:

**Pull request merged**